packages/vrm-viewer/src/vrm-viewer.ts

```diff
@@ -0,0 +1,14 @@
+import { VRMMixin } from './features/vrm-mixin';
+import { ModelViewerElement } from '@google/model-viewer/src/model-viewer';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const VRMViewerElement = VRMMixin(ModelViewerElement);
```

Comment:
not work with [`<model-viewer-effects />`](https://modelviewer.dev/examples/postprocessing/index.html) yet. I think it have to be fixed upstream.